docs/reference/admin/layouts/slots.mdx

```diff
@@ -3,51 +3,59 @@ title: Slots
 ---
 
 import CodeBlock from '@theme/CodeBlock'
-import _components_SlotsSource from '!!raw-loader!./_Example/components/Slots.ts'
+import _components_SlotsSource from '!!raw-loader!./_Example/components/Slots.tsx'
 import _pages_example_Source from '!!raw-loader!./_Example/pages/example.tsx'
 
 # <span className="version">Interface 1.2+</span> Slots
 
+**Use Slots to render contents of the current route to the other areas of the layout.**
```

Comment:
maybe current page? or rephrease it somehow to not mention "the current route"?

Reply:
Done